src/core/server/config/schema/types/string_type.ts

```diff
@@ -50,6 +55,8 @@ export class StringType extends Type<string> {
         return `value is [${value}] but it must have a minimum length of [${limit}].`;
       case 'string.max':
         return `value is [${value}] but it must have a maximum length of [${limit}].`;
+      case 'string.hostname':
+        return `value is [${value}] but it must be a valid hostname.`;
```

Comment:
__note:__ couldn't think of a better wording here, happy to change to anything that sounds better though.

Reply:
Seems fine to me

Reply:
Maybe mention RFC1123 so people have a hint about what is invalid about their hostname?

Reply:
Good idea :heavy_check_mark: